src/rewriter.rs

```diff
@@ -347,6 +347,42 @@ fn extract_source_map<R: Read>(
     }
 }
 
+pub fn generate_prefix_stmts(csi_methods: &CsiMethods) -> Vec<Stmt> {
```

Comment:
Also, it could be moved to an specific file not to add more code in `rewriter.rs`...